walletconnect/.env.example

```diff
@@ -0,0 +1,2 @@
+JSON_RPC_URL="YOUR_VALIDATION_CLOUD_API_ENDPOINT"
```

Comment:
When I ran the code I was getting an error about the walletconnect project id being undefined. It looks like you need to add the prefix VITE before each env variable for 

`import.meta.env.VITE_ENV_VARIABLE` to work as expected.

Reply:
yes missed to change this in the sample file! good spot, thanks!